addons/pos_restaurant/models/pos_config.py

```diff
@@ -131,6 +131,9 @@ def load_onboarding_restaurant_scenario(self, with_demo_data=True):
             'record': config,
             'noupdate': True,
         }])
+        if default_preset:
+            # Ensure the "Presets" menu is visible when installing the restaurant scenario
+            self.env.ref("point_of_sale.group_pos_preset").implied_by_ids |= self.env.ref("base.group_user")
```

Comment:
```suggestion
        if default_preset:
            # Ensure the "Presets" menu is visible when installing the restaurant scenario
            config._get_group_pos_user()._apply_group(self.env.ref("point_of_sale.group_pos_preset"))
```
Can you try with this ?

Reply:
Hello, 
https://github.com/odoo/odoo/blob/bacbb2380611f436c2dd59b9600a74702faf2e6a/odoo/addons/base/models/res_groups.py#L273-L278

Based on my observation, toggling the `group_pos_preset` boolean affects `implied_by_ids`. The current method only adds the group to `implied_ids`, which doesn't impact menu visibility, and no other method in this file appears to handle it.

Thanks have a nice day :innocent:

Reply:
Yeah you're right